Facilitate development from within docker

hblanks opened the pull request:
# Description

Make it easier to run the API, command line scrapes, and tests from docker.

## Type of change

- [x] :sparkles: New feature

# How Has This Been Tested?

As documented in the README.

# Checklist:

- [x] My code follows the style guidelines of this project (pep8 AND pyflakes)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] If needed, I changed related parts of the documentation
- [ ] I included tests in my PR
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] If my PR aims to fix an issue, I referenced it using `#(issue)`

hblanks commented:
FYI, @SamDepardieu  - still one thing left but will try to keep up your good example of incremental pushes / feedback

hblanks commented:
OK! Ready for review.

SamDepardieu commented:
The api.py file was (once) meant to provide users with an easy way to run and monitor spiders. Short story, it wa s a replacement for scrapyd, and unbelievable as it is, it works!
The thing is, we're now running things in AWS, and we're likely to use something like Airflow or whatever framework to connect to AWS and run spiders. We can probably suppose that future users who don't want to run it using AWS can find a way to expose it as they want.

I'd like to hear your thoughts about this, do you think we should keep it (and update it then) or should we drop this?

SamDepardieu commented on Makefile:
```diff
@@ -0,0 +1,15 @@
+
+.PHONY: image
+image:
+	docker build -t $(IMAGE):$(VERSION) .
```
Should we also tag it as `latest` here? We're using the `latest` tageed version in AWS

hblanks commented:
@SamDepardieu  - OK! I have:

1. Moved api.py into attic/ and added a README,
2. Removed the last references to the periodic docker image
3. Added a tag for `latest` since, yes, it seems like the thoughtful thing to do. Thanks for the note on it!
